editor.planx.uk/src/pages/Team/index.tsx

```diff
@@ -68,13 +94,31 @@ const Team: React.FC = () => {
 
   const route = useCurrentRoute();
 
+  const [showCardGrid, toggleCardGrid] = useStore((state) => [
+    state.showCardGrid,
+    state.toggleCardGrid,
+  ]);
```

Comment:
This doesn't need to be tracked in both React state and the Zustand store - doing both could well lead to bugs. If we just get the functions/values we need from Zustand we could call them directly.

```ts
const [{ id: teamId, slug }, canUserEditTeam, getFlows, isTrial, showCardGrid, toggleCardGrid] = useStore(
  (state) => [
    state.getTeam(),
    state.canUserEditTeam,
    state.getFlows,
    state.teamSettings?.isTrial,
    state.showCardGrid,
    state.toggleCardGrid,
  ],
  ```